pkg/stanza/operator/parser/container/config.go

```diff
@@ -39,6 +40,7 @@ func NewConfigWithID(operatorID string) *Config {
 		ParserConfig:            helper.NewParserConfig(operatorID, operatorType),
 		Format:                  "",
 		AddMetadataFromFilePath: true,
+		FilePathPattern:         "",
```

Comment:
Shall we use the `/var/log/pods` respective pattern as the default one directly here?

Reply:
I can't, because the default pattern is defined in a different file: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/d50cb71630064e614ccd8e98caf27d9f81e6cfb2/pkg/stanza/operator/parser/container/parser.go#L34)